### PR TITLE
lua cleanup a few job abilities

### PR DESCRIPTION
**Pull request by isotor**

### CODE REVIEW

File: scripts/globals/abilities/chi_blast.lua [hunk 1]
```diff
@@ -29,4 +29,8 @@ function onUseAbility(player,target,ability)
     target:updateEnmityFromDamage(player,dmg)
     player:delStatusEffect(dsp.effect.BOOST)
     return dmg
+
+	if dmg > 0 and target(isAsleep) then
```
Comment (TeoTwawki) [inline scripts/globals/abilities/chi_blast.lua]: the return above means this will never be hit.

Comment (isotor) [reply scripts/globals/abilities/chi_blast.lua]: Woops, it's early plz forgive

**Comment by ibm2431** [comment]
[target:takeDamage in core should already be accounting for this](https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L7451)

**Comment by isotor** [comment]
Ah, fair point. Removed, but I figure may as well leave the PR open just for the free lua cleanup.

**Pull request merged**